src/LuaPrinter.ts

```diff
@@ -129,8 +129,11 @@ export class LuaPrinter {
                 header += `require("lualib_bundle");\n`;
             }
             // Inline lualib features
-            else if (this.options.luaLibImport === LuaLibImportKind.Inline && luaLibFeatures.size > 0)
-            {
+            else if (
+                (this.options.luaLibImport === undefined ||
```

Comment:
We generally put operators first on the next line, instead of at the end of lines, but maybe here an intermediate `const luaLibImport = this.options.luaLibImport || LuaLibImportKind.Inline;` would be a better solution.

Reply:
What do you think about using switch there?

Reply:
Not really a fan of switch since we are only checking one value or undefined, plus there is still the size check.